config/jobs/kubernetes-sigs/gateway-api-inference-extension/inference-extension-periodics-main.yaml

```diff
@@ -0,0 +1,35 @@
+periodics:
+  - interval: 12h
+    name: periodic-gateway-api-inference-extension-test-unit-main
+    cluster: eks-prow-build-cluster
+    extra_refs:
+      - org: kubernetes-sigs
+        repo: gateway-api-inference-extension
+        base_ref: main
+        path_alias: sigs.k8s.io/gateway-api-inference-extension
+    annotations:
+      testgrid-dashboards: sig-apps
```

Comment:
Is this sig-networking or sig-apps? This should be the owning sig for the repo.

Reply:
sig net. Thanks for this! I moved it under gateway-api's dash.